### ISSUE TITLE
Support config query

### ISSUE THREAD

**Issue by hallyn** [openSUSE/umoci#218]
There is a 'umoci config' command, which lets you set --config.entrypoint and --config.cmd.  But near as I can tell you can only set, not query the current value. It would be nice to be able to query that value.

**Comment by hallyn** [comment]
(For now I will use 'umoci raw runtime-config --image oci:tag xx.json' and parse the intermediate file, but that's not ideal)

**Comment by cyphar** [comment]
Yeah. Ideally, `umoci stat` would have a better interface that would give you that sort of information. I haven't worked on `umoci stat` much (I mainly wrote it so that we could use it to test that the history was correctly modified) but it would be even cooler if we could query things with high granularity using `umoci stat`.
